repose-aggregator/components/filters/translation-filter/src/test/java/org/openrepose/filters/translation/TranslationFilterMethodsTest.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * _=_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=
+ * Repose
+ * _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+ * Copyright (C) 2010 - 2015 Rackspace US, Inc.
+ * _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=_
+ */
+package org.openrepose.filters.translation;
+
+import org.apache.commons.io.IOUtils;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.openrepose.commons.utils.servlet.http.HandleRequestResult;
+import org.openrepose.commons.utils.servlet.http.HttpServletRequestWrapper;
+import org.openrepose.commons.utils.servlet.http.HttpServletResponseWrapper;
+import org.openrepose.commons.utils.servlet.http.ResponseMode;
+import org.openrepose.core.services.config.ConfigurationService;
+import org.openrepose.filters.translation.config.*;
+import org.springframework.mock.web.MockFilterConfig;
+import org.springframework.mock.web.MockHttpServletRequest;
+import org.springframework.mock.web.MockHttpServletResponse;
+import org.xml.sax.SAXException;
+
+import java.io.IOException;
+
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.springframework.http.HttpHeaders.ACCEPT;
+import static org.springframework.http.MediaType.APPLICATION_XML_VALUE;
+import static org.xmlunit.matchers.CompareMatcher.isSimilarTo;
+
+public class TranslationFilterMethodsTest {
+    private static byte[] contentRemoveMe;
+
+    private TranslationFilter filter;
+    private MockHttpServletRequest mockRequest;
+    private MockHttpServletResponse mockResponse;
+    private HttpServletRequestWrapper httpServletRequestWrapper;
+    private HttpServletResponseWrapper httpServletResponseWrapper;
+
+    @BeforeClass
+    public static void setupSpec() throws Exception {
+        contentRemoveMe = IOUtils.toByteArray(TranslationFilterMethodsTest.class.getResourceAsStream("/remove-me-element.xml"));
+    }
+
+    @Before
+    public void setup() throws Exception {
```

Comment:
Yep. I agree this is not the best, but I was making them all behave similarly for continuity.